metadata.json

```diff
@@ -10,11 +10,11 @@
   "dependencies": [
     {
       "name": "puppetlabs-stdlib",
-      "version_requirement": ">= 4.2.0"
+      "version_requirement": ">= 4.3.0"
```

Comment:
Oops - my mistake....

https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/CHANGELOG.md#2014-05-08---release---420

I thought `delete_undef_values` was released in 4.3, it was actually 4.2 so I'll revert this change.